src/Microsoft.AspNetCore.Sockets/Transports/LongPollingTransport.cs

```diff
@@ -32,6 +32,9 @@ public LongPollingTransport(ReadableChannel<Message> application, ILoggerFactory
                 // This is for cases when the client reconnects see issue #27
                 if (!await _application.WaitToReadAsync(context.RequestAborted))
                 {
+                    // If the completion is faulted or cancelled, we want to manifest that error
+                    _application.Completion.GetAwaiter().GetResult();
```

Comment:
Right now, if the app completes the channel with an error, that error vanishes into the ether. I can log it or rethrow it. Here, I chose to rethrow it. If you think logging would be better, then we can do that instead.